apps/verticalhome/js/app_manager.js

```diff
@@ -7,7 +7,7 @@
 (function(exports) {
 
   function AppManager() {
-    var grid = document.querySelector('gaia-grid');
+    var grid = document.querySelector('gaia-grid-rs');
```

Comment:
If I understand correctly, the gaia-grid is now only used for the more apps section, right? So maybe this entire module should be renamed to MoreAppsManager or something.  (If you do a git mv you can rename a file without making the entire file look like new code)